src/index.js

```diff
@@ -30,8 +32,26 @@ export async function run() {
       core.debug("Generated SSH-Key successfully")
     }
 
+    let newSessionExtra = ""
+    if (core.getInput("limit-access-to-actor") === "true") {
+      const actor = github.context.actor
+      const octokit = new Octokit()
+
+      const keys = await octokit.users.listPublicKeysForUser({
+        username: actor
+      })
+      if (keys.data.length < 1) {
+        throw new Error(`No public SSH keys registered with ${actor}'s GitHub profile`)
+      }
+      const sshPath = path.join(process.env.HOME, ".ssh")
```

Comment:
```suggestion
      const sshPath = path.join(os.homedir(), ".ssh")
```
Seems to be the fix Windows.

Reply:
D'oh!